nested.just

```diff
@@ -19,20 +19,20 @@ simulate whichSafe hdPath='0':
   #!/usr/bin/env bash
   bundlePath=${taskPath}/${bundleName}.json
   echo "getting signer address..."
-  if [ t"{{whichSafe}}" == t"foundation" ]; then
```

Comment:
This diff needs to be undone

Reply:
@zchn where was this `t"` syntax from? could not find it in just or bash docs

Reply:
I was able to get the command to work as expected with the diff included in this PR. Without it, it was not hitting either `if` branch and resulting in empty `safe` and `signer` variables